### DISCUSSION TITLE
Help

### DISCUSSION THREAD

**Discussion by Mraul44** [anidl/multi-downloader-nx#602] [Q&A]
Hello!! I was looking for a Crunchyroll downloader since the one I used no longer works. The truth is that I am new to this, so could you make a video or tell me more or less the steps to follow to be able to download the videos from crunchyroll, please.

**Comment by Jaynator495** [comment]
I suggest trying the GUI to start with. Then look into dumping a L3 CDM and stick it in the widevine folder. Finally download mp4decrypt and add it to your path. (or specify the path in bin-path). If all is done right it should work. If you need more help I encourage you to join the discord
